Article/Article.js

```diff
@@ -3,24 +3,24 @@
 class Article {
   constructor(element) {
     // assign this.element to the passed in article element
-    this.element;
+    this.element = element; //DOM
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.element.querySelector(".expandButton").innerHTML = "expand";
```

Comment:
Based on the comments, this should be:
`this.expandButton = this.element.querySelector(".expandButton");`
`this.expandButton.innerHTML = "expand";`

In the way you did it, I'm not sure what would be stored in `this.expandButton`. You should `console.log` it to see if the results are the same under the hood. If not, then you need to do it the way I demonstrated in case you end up wanting to use `this.expandButton` for other things elsewhere.